Remora.Sdk/Sdk/Sdk.targets

```diff
@@ -11,6 +11,9 @@
 
         <!--suppress MsbuildTargetFrameworkTagInspection -->
         <TargetFrameworks Condition="'$(OutputType)' == 'Exe'">$(ExecutableFrameworks)</TargetFrameworks>
+
+        <!--suppress MsbuildTargetFrameworkTagInspection -->
+        <TargetFrameworks Condition="'$(TargetNetStandard)' == true">netstandard2.1;$(TargetFrameworks)</TargetFrameworks>
```

Comment:
Do I need the suppress here?

Reply:
As it is, you're adding netstandard2.1 to both executable and library targets, which won't work. You should do this below `<TargetFrameworks>$(LibraryFrameworks)</TargetFrameworks>`, and use `$(LibraryFrameworks)` inside the value instead.

You technically can't build an executable target netstandard, even though it works in some cases.